### ISSUE TITLE
false positive "no-member"

### ISSUE THREAD

**Issue by wmpauli** [PyCQA/pylint#2888]
Hi, I want to preface this by saying that I am not sure this is a bug, but could be the results of non-standard implementation of the opencv module. Thank you.

### Steps to reproduce
1. install opencv-python = 4.1.0.25
2.:
```
import cv2
videocap = cv2.VideoCapture('filename.mp4')
```

### Current behavior

I get a "no-member" highlighting for VideoCapture class, but the code runs as expected (the member exists).

Here is the code of the ```__init__.py``` for cv2, in case that's helpful.

```
import importlib

from .cv2 import *
from .data import *

# wildcard import above does not import "private" variables like __version__
# this makes them available
globals().update(importlib.import_module('cv2.cv2').__dict__)

```

### Expected behavior

no false-positive highlighting of VideoCapture class

### pylint --version output

> pylint 2.3.1
> astroid 2.2.5
> Python 3.6.8 |Anaconda, Inc.| (default, Dec 30 2018, 01:22:34) 
> [GCC 7.3.0]

**Comment by PCManticore** [comment]
@wmpauli `cv2` seems to be a C extension, which explains why `pylint` cannot find the `VideoCapture` class. There's not much `pylint` can do about it but if you use `--extension-pkg-whitelist=cv2` you can force `pylint` to build ASTs from the live `cv2` module, that is pylint is going to import it in order to construct its internal representation, rather than relying on static analysis to provide that representation. This should fix the issue you have.

**Comment by wmpauli** [comment]
Thank you, @PCManticore
